### ISSUE TITLE
yolo-pose custom train question

### ISSUE THREAD

**Issue by tonyskypc** [TexasInstruments/edgeai-yolov5#87]
git code about yolo-pose,use coco 2017 data train,detect result keypoint out of range.
![Uploading a.jpg…]()

**Comment by ZhouKai90** [comment]
I come across the same problem.

**Comment by go2carter** [comment]
same. Has anybody figured out what it is or how to resolve it?

**Comment by nomaad42** [comment]
Still no one solved it?

**Comment by zay95** [comment]
same issue. Train labels is normal, but predict labels is wrong using pretrained model yolov7-w6-person.pt. And image size is 640, adam optimizer  lr  0.001. But loss doesn't add constant value in time yesterday.

![000000012280_pred](https://user-images.githubusercontent.com/33301898/225561567-860829ad-c002-4a4c-800f-3e613133a0bd.jpg)
